keps/sig-node/1539-hugepages/README.md

```diff
@@ -546,6 +560,67 @@ locality guarantees as a feature of QoS.  In particular, pods in the
     - Test case will be added to cri-tools to be used in CRI runtime' test(CI).
     - here: https://github.com/kubernetes-sigs/cri-tools
 
+## Test Plan for HugePageStorageMediumSize
+
+- Promote existing HugePages E2E tests to conformance
+
+## Production Readiness Review Questionnaire
+### Monitoring requirements
+
+* **How can an operator determine if the feature is in use by workloads?**
+An operator could audit pods that use hugepages resources.
```

Comment:
How?

Reply:
The way it's shown in the documentation: https://kubernetes.io/docs/tasks/manage-hugepages/scheduling-hugepages/

Reply:
I've added some wording around it. Please review.